example_10/test/test_rrbot_launch.py

```diff
@@ -50,11 +49,8 @@
 
 # This function specifies the processes to be run for our test
 # The ReadyToTest action waits for 15 second by default for the processes to
-# start, if processes take more time an error is thrown. We use decorator here
-# to provide timeout duration of 20 second so that processes that take longer than
-# 15 seconds can start up.
+# start, if processes take more time an error is thrown.
```

Comment:
Is this statement still true? I don't see any evaluation of checking if it takes longer or not

Reply:
yes, this is part of the launch_testing
https://github.com/ros2/launch/blob/f3e9cfebcfa75c82299bb31674355a29454489fa/launch_testing/test/launch_testing/examples/ready_to_test_action_timeout_test.py#L25-L32